crates/web-plugins/didcomm-messaging/protocol-plugins/handler/src/plugins.rs

```diff
@@ -0,0 +1,20 @@
+
+#[cfg(feature = "plugin-index")]
+pub(crate) mod index;
+
+
+use lazy_static::lazy_static;
+use std::sync::{Arc, Mutex};
+
+use plugin_api::Plugin;
+
+lazy_static! {
+    pub(crate) static ref PLUGINS: Vec<Arc<Mutex<dyn Plugin + 'static>>> = vec![
+        #[cfg(feature = "protocol-forward")]
+        // Arc::new(Mutex::new(index::IndexPlugin {})),
+        #[cfg(feature = "protocol-pickup")]
+        // Arc::new(Mutex::new(did_endpoint::plugin::DidEndpoint {})),
+        #[cfg(feature = "protocole-mediator-coordination")]
```

Comment:
it's preferable using underscores here instead of hyphens

Reply:
The hyphens are acceptable; we can proceed with as it is